api/src/main/resources/messages.properties

```diff
@@ -65,6 +65,7 @@ error.dwr=A javascript error has occurred:
 error.dwr.hide=Hide error
 error.outOfRange.low.obs.valueNumeric=This value is too low to be valid for this question
 error.outOfRange.high.obs.valueNumeric=This value is too high to be valid for this question
+error.outOfRange.criteria.not.match.obs.valueNumeric=The concept value does not match criteria
```

Comment:
Can you include some more details about the criteria upper or lower bound values to make the error message more meaningful?

Reply:
What do you mean by upper and lower bound?

Reply:
For instance, `Expected a value between 0 and 10`

Reply:
What about other parts of the criteria? Do you suggest that we should check every part of criteria and update this message dynamically?